semantic/lookup.go

```diff
@@ -11,12 +11,21 @@ import (
 
 var stdlibTypeEnvironment = TypeEnvMap(fbsemantic.GetRootAsTypeEnvironment(libflux.EnvStdlib(), 0))
 
-// LookupBuiltInType returns the type of the builtin value for a given
+type envKey struct {
+	Package string
+	Prop    string
+}
+
+// LookupBuiltinType returns the type of the builtin value for a given
 // Flux stdlib package. Returns an error if lookup fails.
-func LookupBuiltInType(pkg, name string) (MonoType, error) {
-	var table flatbuffers.Table
-	prop := stdlibTypeEnvironment[pkg][name] // query environment map for prop
+func LookupBuiltinType(pkg, name string) (MonoType, error) {
+	key := envKey{
+		Package: pkg,
+		Prop:    name,
+	}
+	prop := stdlibTypeEnvironment[key] // query environment map for prop
```

Comment:
You should check if the prop was missing so an error can be returned.

```
prop, ok := stdlibTypeEnvironment[key] 
if !ok {
    // return error
}
```

Reply:
Yeah that's a good idea. I figured it didn't matter since we are panicking in most places where we use this function, but you're right its more consistent to return an error here.